orca/reconstruct.py

```diff
@@ -0,0 +1,14 @@
+from xarray import open_mfdataset, open_dataset
+
+
+def reconstruct_dataset(temp_files, outfile):
+    temp_paths = [temp.name for temp in temp_files]
+    dataset = (
+        open_mfdataset(temp_paths, combine="nested", concat_dim="time")
+        if len(temp_files) > 1
+        else open_dataset(temp_paths[0])
+    )
+    (temp.close for temp in temp_files)
+    dataset.to_netcdf(outfile)
+
+    return outfile
```

Comment:
One thing we should consider here is that the `to_netcdf` method is pretty slow. I think it is actually the main bottleneck of the process and something I am learning more about in issue #13. Looking briefly at the docs you linked, I don't think we can use `urls` in place of `temp_paths` for `open_mfdataset`. If it is possible we may want to reconstruct directly after splitting to save some time.